pkg/upgrade/rollout.go

```diff
@@ -0,0 +1,60 @@
+package upgrade
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/threefoldtech/zos/pkg/environment"
+)
+
+const defaultRolloutConfigURL = "https://raw.githubusercontent.com/threefoldtech/zos/development_rollout_update/pkg/upgrade/config.json"
+
+// RolloutConfig is the configuration for A/B testing before zos update
```

Comment:
>  A/B testing

whitelist node/farm?

Reply:
please don't use `white/black` lists in code. only use allowed/disallowed lists